plone/recipe/codeanalysis/README.rst

```diff
@@ -85,3 +85,16 @@ Then the git pre-commit hook has been installed::
 
     >>> 'install git pre-commit hook.' in buildout_output_lower
     True
+
```

Comment:
I would say this is an implementation detail that does not need to be covered in the docs. Therefore I would suggest to rewrite the test into a regular integration test.